### DISCUSSION TITLE
Track taxonomy archive pages, like categories and tags

### DISCUSSION THREAD

**Discussion by dannyvankooten** [ibericode/koko-analytics#212] [Ideas]
See #62 and #45

**Comment by Takkster** [comment]
I think this is a must-have. Clients don't discern pages or posts (with IDs) from taxonomy archives and the likes. They just ask, why is my "insert category here" not showing up in the stats. Meanwhile, I'll try the solution suggested in https://github.com/ibericode/koko-analytics/issues/62

**Comment by nickbohle** [comment]
Please include all archives.

- Categories
- Tags
- Time
- Author(s)
- Post formats (slug is /type)
- Custom Post Types

Thanks.

**Comment by dannyvankooten** [comment]
✔️ since Koko Analytics version 2.0!

https://www.kokoanalytics.com/2025/08/27/koko-analytics-version-2-is-here/
